registry-resolver/src/lib.rs

```diff
@@ -59,12 +46,85 @@ impl ssi::DIDResolver for RegistryResolver {
 
 #[cfg(test)]
 mod tests {
-    use crate::DID_METHOD;
+    use ssi::DIDResolver;
 
-    #[test]
-    fn test_create() -> Result<(), String> {
-        assert!(false);
-        Ok(())
+    use crate::{
+        registry_client::registry::CreateResponse, registry_client::MockRegistryClient,
+        RegistryResolver,
+    };
+
+    fn create_did_doc(did: String) -> serde_json::Value {
+        return serde_json::json!({
```

Comment:
should we store this in a test json file and do an fs read?

Reply:
@mkatychev What's the benefit of that? I feel like storing it in a separate file reduces the context of it which increases the likelihood of misuse

Reply:
that's a good point I suppose I was thinking of it in terms of reusable test dummy data such as the directory [monetae/core has](https://github.com/monetae/core/tree/develop/integration/data). I see your point about cognitive load.